crates/matrix-sdk-ui/src/timeline/event_item/content/message.rs

```diff
@@ -373,20 +379,69 @@ impl RepliedToEvent {
 
         debug!(event_type = %event.event_type(), "got deserialized event");
 
-        let Some(AnyMessageLikeEventContent::RoomMessage(c)) = event.original_content() else {
-            warn!("can't get details, event is redacted or not a RoomMessage");
-            return Err(TimelineError::UnsupportedEvent);
-        };
+        let content = match event.original_content() {
+            Some(content) => match content {
+                AnyMessageLikeEventContent::RoomMessage(c) => {
+                    // Assume we're not interested in reactions in this context.
```

Comment:
It might make sense to explain why we're not interested.

Reply:
Good point, edited with this comment:

```rust
// Assume we're not interested in reactions in this context: this is
// information for an embedded (replied-to) event, that will usually not
// include detailed information like reactions.
```